### ISSUE TITLE
How to open 'Oh my Git' on Linux?

### ISSUE THREAD

**Issue by gdwhittaker94** [git-learning-game/oh-my-git#180]
Hi there. 

I would like to try 'Oh my Git' and I'm using a Linux OS. 

I've downloaded the folder, clicked on the file 'oh-my-git' and my laptop is unable to access it. It tries to open it with a program called 'Anki', and apart from that it doesn't give me any other options or programs to open the game with.  

Using the terminal, I've created a folder, put the files in the folder, accessed the folder via the terminal, but ... then what? What command do I need to run the game?

Thanks in advance!

**Comment by bleeptrack** [comment]
Hey,
thanks a lot for trying 'Oh my Git'! Just to make sure: you downloaded the .zip from https://blinry.itch.io/oh-my-git, correct?

On your terminal, go to the folder you created which contains everything inside the .zip (that should be two files).
then run 
`./oh-my-git `
(including the dot and slash - that's a shorthand command to execute a file).
Let me know if that worked for you :)

**Comment by gdwhittaker94** [comment]
Hi Bleeptrack,

Ah ok, I saw ./ on the website but I didn't realise it was a shorthand command! 

So I have followed your instructions, changing directory to the directory that contains the oh-my-git files, run that command and I get a "permission denied" message in the terminal. 

What now?

![Screenshot 2023-07-17 11 36 59](https://github.com/git-learning-game/oh-my-git/assets/105855731/a450827f-0aef-4937-ac58-ab013caf86e9)
